src/java/main/org/apache/zookeeper/server/quorum/QuorumCnxManager.java

```diff
@@ -857,15 +869,15 @@ public void run() {
                         self.recreateSocketAddresses(self.getId());
                         addr = self.getElectionAddress();
                     }
-                    LOG.info("My election bind port: " + addr.toString());
+                    LOG.info("My election bind port: " + formatInetAddr(addr));
                     setName(addr.toString());
```

Comment:
using  `toString` can still distinguish ipv4 from ipv6 by looking at the ip style although without `[] ` 
if `formatInetAddr(addr)` is really needed, can you make sure all the places which log `InetSocketAddress` are covered?

Reply:
Good point - I'll remove formatInetAddr(addr) from the log